src/components/FooterContainer/styles.js

```diff
@@ -0,0 +1,126 @@
+export const footer = {
+  display: "flex",
+  justifyContent: "center",
+  backgroundColor: "#F5FBFC",
+  color: "#333",
+  width: "100%",
+  flexDirection: "column",
+  alignItems: "center",
+};
+
```

Comment:
This is an interesting approach, but finally these styles becomes inline styles, which are more specific than other and it will be difficult to beat them if necessary.